lib/smart_proxy_container_gateway/container_gateway_api.rb

```diff
@@ -6,23 +6,29 @@
 require 'smart_proxy_container_gateway/container_gateway'
 require 'smart_proxy_container_gateway/container_gateway_main'
 require 'smart_proxy_container_gateway/foreman_api'
+require 'smart_proxy_container_gateway/dependency_injection'
 require 'sqlite3'
+require 'sequel'
```

Comment:
Aren't these 2 more logical in the `Main` file?

Reply:
We do some direct DB calls in here, so I have to check.

Reply:
Actually I suppose that doesn't matter since the api file requires the main file.